app/models/manageiq/providers/amazon/storage_manager/s3/cloud_object_store_container/operations.rb

```diff
@@ -0,0 +1,7 @@
+module ManageIQ::Providers::Amazon::CloudManager::CloudObjectStoreContainer::Operations
```

Comment:
Yeah, because of such small files I'm not a big fan of spreading the code across modules.

So, if you like, you can inline this here in the aws gem 😄  

https://blog.codeship.com/good-module-bad-module/